src/System.Runtime/tests/System/DateTimeTests.cs

```diff
@@ -896,5 +966,153 @@ public static void InvalidDateTimeStyles()
+            yield return new object[] { "99", "dd", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123", "dd", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "1", "mm", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "99", "mm", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123", "mm", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "1", "ss", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "99", "ss", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123", "ss", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "1", "MM", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "99", "MM", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "Fep", "MMM", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "Jantember", "MMMM", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "123", "YY", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "12345", "YYYY", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "1", "HH", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "99", "HH", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123", "HH", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "1", "hh", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "99", "hh", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123", "hh", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "1", "ff", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123", "ff", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "123456", "fffff", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "1234", "fffff", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+
+            yield return new object[] { "AM", "t", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "PM", "t", CultureInfo.InvariantCulture, DateTimeStyles.None, };
+            yield return new object[] { "PM", "ttt", CultureInfo.InvariantCulture, DateTimeStyles.None, };
```

Comment:
trailing commas on all these initializers?

Reply:
Just an outcome of copy and paste :)